src/xrCore/Threading/TaskManager.cpp

```diff
@@ -275,9 +275,7 @@ void TaskManager::WakeUpIfNeeded()
                 continue;
             if (worker->sleeps.load(std::memory_order_relaxed))
             {
-                // Acquire semantic guarantees that event won't be set
-                // earlier than steal_from will be assigned
-                worker->steal_from.store(steal_from, std::memory_order_acquire);
+                worker->steal_from.store(steal_from);
```

Comment:
Well, I'm not sure, but maybe `Set()` function call just won't reordered, so we can just relax... But I'm not sure.
```suggestion
                worker->steal_from.store(steal_from, std::memory_order_relaxed);
```

Reply:
Well, i'm not sure neither, but if it won't reorder anyway, should we even specify any order manually? We can just leave the default strict ordering.

Reply:
The lesser restrictions – the better for performance. Except, that it should always be safe. It is safe to lower the restrictions here.